packages/components/src/components/Select/Select.stories.tsx

```diff
@@ -0,0 +1,253 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconIcon } from '@frontify/fondue-icons';
+import { type Meta, type StoryObj } from '@storybook/react';
+import { useState } from 'react';
+
+import { Button } from '../Button/Button';
+
+import { SelectCombobox } from './Combobox';
+import { Select, SelectInput } from './Select';
+import { SelectItem, SelectItemGroup } from './SelectItem';
+import { SelectSlot } from './SelectSlot';
+
+type Story = StoryObj<typeof meta>;
+const meta: Meta<typeof SelectInput> = {
+    component: SelectInput,
+    subcomponents: {
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Combobox': SelectCombobox,
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Slot': SelectSlot,
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Group': SelectItemGroup,
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Item': SelectItem,
+    },
+    tags: ['autodocs'],
+    parameters: {
+        status: {
+            type: 'in_progress',
+        },
+    },
+    args: {
```

Comment:
let's add a `placeholder` else the select looks "broken"

Reply:
added